inc/saiswitch.h

```diff
@@ -237,12 +237,20 @@ typedef enum _sai_switch_attr_t
     SAI_SWITCH_ATTR_START,
 
     /**
-     * @brief The number of ports on the switch
+     * @brief Number of active ports on the switch
      *
      * @type sai_uint32_t
      * @flags READ_ONLY
      */
-    SAI_SWITCH_ATTR_PORT_NUMBER = SAI_SWITCH_ATTR_START,
+    SAI_SWITCH_ATTR_NUMBER_OF_ACTIVE_PORTS = SAI_SWITCH_ATTR_START,
```

Comment:
for backward compatibility, please follow commit https://github.com/opencomputeproject/SAI/commit/3cc57f24b0d5df1b0e41abc5450cbbe2592352bf